arm-software/linux/docs/GettingStarted.md

```diff
@@ -333,6 +333,11 @@ generates  portable output suitable for any Armv8-A-based target.
   `-mcpu=<target>` where `<target>` is the target processor that you will run
   the application on.
 
+### Standards support
+
+For information on C, C++, and Fortran language support in ATfL, see the [Standards Support page](StandardsSupport).
```

Comment:
shouldn't it be StandardsSupport.md and OpenMPSupport.md?

Reply:
Yes. Changed now.